src/Sylius/Behat/Page/Shop/Cart/SummaryPage.php

```diff
@@ -214,18 +166,12 @@ public function hasItemWithOptionValue($productName, $optionName, $optionValue)
         return $optionValue === $optionValueElement->getText();
     }
```

Comment:
I think that you should replace UI classes in this line too : `$selector = sprintf('.sylius-product-options > .item[data-sylius-option-name="%s"]', $optionName);`